holmes/core/toolset_manager.py

```diff
@@ -111,9 +114,13 @@ def _list_all_toolsets(
                 if any(tag in toolset_tags for tag in toolset.tags)
             }
 
+        # Apply global transformer configurations to all toolsets
+        final_toolsets = list(toolsets_by_name.values())
+        self._apply_global_transformer_configs(final_toolsets)
```

Comment:
Honeslty i don't think it is a good idea to enable it by default to all toolsets even if there wasn't trnasformer configure for it. Can you give an example for global transformer that you think we would like to enable on all toolsets?

Reply:
Now only ones with transfomer defined will be merged with global transfomers.